scripts/image-builder/configure.sh

```diff
@@ -18,6 +18,8 @@ sudo dnf install -y git osbuild-composer composer-cli ostree rpm-ostree \
     createrepo yum-utils selinux-policy-devel jq wget lorax rpm-build \
     containernetworking-plugins expect
 
+sudo sed -i 's,dist/rhel9/9.3,beta/rhel9/9,g' /usr/share/osbuild-composer/repositories/rhel-93.json
```

Comment:
We probably need to run this command only if the file exists and we're working on a beta OS, right?

Reply:
Yup, it's just draft PR now, needed to check how things will work in CI compared to local env